io.github.jpochyla.Psst.yaml

```diff
@@ -0,0 +1,38 @@
+app-id: io.github.jpochyla.Psst
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+command: psst-gui
+finish-args:
+- "--share=ipc"
+- "--share=network"
+- "--socket=fallback-x11"
+- "--socket=wayland"
+- "--device=dri"
```

Comment:
Does it really need video drivers? GTK 3 in general doesn't.

Reply:
I have no idea